src/battle_script_commands.c

```diff
@@ -11041,6 +11042,7 @@ static void Cmd_pickup(void)
                     SetMonData(&gPlayerParty[i], MON_DATA_HELD_ITEM, ITEM_HONEY);
```

Comment:
Same thing here.

Reply:
Isn't it already being checked by line 11031?
![image](https://user-images.githubusercontent.com/2904965/74122460-09410100-4baa-11ea-82c4-43bedfd09fa6.png)

Reply:
It should be:
```
                heldItem = ITEM_HONEY;
                SetMonData(&gPlayerParty[i], MON_DATA_HELD_ITEM, &heldItem);
```

It will throw a compiling error if you write it that way. It needs a pointer as 3rd arg.